jobconfig.toml

```diff
@@ -73,8 +75,8 @@
 [CARGO_TECHNICIAN]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 2
-"Total Positions" = 2
+"Spawn Positions" = 5
```

Comment:
Icebox actually only has 3, not sure how to feel about this change. 5 total job slots are fine.

Reply:
icebox/biodome 3, northstar 4, meta 5

Reply:
Reducing cargo technician spawn positions to 3 to prevent problems. If testing with another commit proves that this spawn position increase is okay, then will raise it back to the intended 5.